src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+public class Tagging {
+	private ArrayList<String> addTag = new ArrayList<String>();
+	private ArrayList<String> deleteTag = new ArrayList<String>();
+	
+	public ArrayList<String> getAddTag(){
+		return addTag;
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.